app/src/main/java/com/mapbox/vision/examples/activity/ar/ArNavigationActivity.kt

```diff
@@ -43,15 +43,13 @@ class ArNavigationActivity : AppCompatActivity(), RouteListener, ProgressChangeL
     companion object {
         private var TAG = ArNavigationActivity::class.java.simpleName
 
-        private const val EXTRA_ROUTE = "Route"
         private const val LOCATION_INTERVAL_DEFAULT = 0L
         private const val LOCATION_INTERVAL_FAST = 1000L
 
-        fun start(context: Activity, directionsRoute: DirectionsRoute) {
-            context.startActivity(
-                Intent(context, ArNavigationActivity::class.java)
-                    .putExtra(EXTRA_ROUTE, directionsRoute)
-            )
+        var directionsRoute: DirectionsRoute? = null
```

Comment:
hope nobody will know that I approved that 😞

Reply:
@korshaknn Lets file a ticket to change this hacky fix?

Reply:
@yunikkk done